api/spec/v1.yaml

```diff
@@ -1614,6 +1675,76 @@ components:
       description: |
         A ParaTime block.
 
+    RuntimeEventList:
+      type: object
+      required: [events]
+      properties:
+        events:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeEvent'
+      description: |
+        A list of runtime events.
+
+    RuntimeEvent:
+      type: object
+      required: [round, type, body]
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block height at which this event was generated.
+          example: *runtime_block_round_1
+        tx_hash:
+          type: string
+          nullable: true
+          description: |
+            Hash of this event's originating transaction
+            Absent if the event did not originate from a transaction.
+          example: *tx_hash_1
+        type: 
```

Comment:
See comment in DB and on RuntimeEventType. We probably want another filter here, `evm_log_name`. I think that one should be a free-form string; as mentioned elsewhere, we cannot enumerate all log names.
Also important, an example value here is "Transfer" (verbatim from the log), not "erc20.Transfer" or something more semantic. Yes the user might get some non-erc20 tokens' logs in response, but so it goes. If this is a problem in practice, we can add _yet another_ filter, "give me only events that come from an EVM contract that is of type X" (where X is the filter value, and is for example ERC20). But that will complicate the DB query, so let's not
do it until/if actually needed.

Reply:
Got it, added!